### PR TITLE
Added topics

### PR DESCRIPTION
**Pull request by germanysources**
Added a reader for topics of github repositories in the class GHTopcis.

In the class GHRepository added a get method for the license (key and name).

### CODE REVIEW

**Comment by bitwiseman** [comment]
@germanysources  
Do you have time to bring this PR up to date?

**Comment by germanysources** [comment]
@bitwiseman 
I will look after it next month. 
What changes did you need except an sucessfull CI-build?

File: src/test/java/org/kohsuke/github/TestTopics.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import org.junit.Assert;
+import org.junit.BeforeClass;
+
+import java.io.*;
+import java.nio.file.*;
+import java.util.*;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class TestTopics extends GHTopics{
```
Comment (bitwiseman) [inline src/test/java/org/kohsuke/github/TestTopics.java]: Switch to using WireMock

**Comment by bitwiseman** [comment]
I'm going to close this for now.  Please ping when you have time to update it.

**Comment by mttjj** [comment]
Any chance this work could be revived? We have a need for using repo topics in our organization of 350+ repos.

**Pull request closed without merging**